ui/src/components/Layout/Menubar.vue

```diff
@@ -420,6 +439,15 @@ const onMenuItemClick = (url: string, isVueLink?: boolean | null) => {
   window.location.assign(link)
 }
 
+const onSelfServiceMenuItemClick = (item: MenuItem) => {
+  if (item.id === 'logout') {
```

Comment:
Are we embedding the form, generating the URL on render, and then roundabout submitting the blank form just to get the POST + redirect? I wonder if we could get the same functionality with a POST `fetch` call + a redirect to the login page.

I'll approve this regardless of my question because I assume it's mimicking how the old logout button worked to reduce integration friction and speed up time to acceptance. 

But maybe we can create a new task as tech debt for the future to create an integrated and shareable logout method that both UIs can use. It might be a good idea to try and consolidate shared functionality over time if we're going to carry both UIs into the future. (So that a future update to the Navbar logout functionality doesn't result in twice as much work to update both implementations, which would be annoying from a dev perspective I think).

Reply:
Yes that's what we're doing. I agree, should be tech debt and create a better approach.